src/core/interaction-creator.ts

```diff
@@ -112,7 +112,7 @@ export class InteractionCreator {
             const basicTypeRepresentationOfRequestBody = getTypeRepresentation(queryElementType, this.sourceFile);
             const exampleRepresentationOfQueryObject = changeObjectRepresentationIntoExample(basicTypeRepresentationOfRequestBody);
             return {
-                query: qs.stringify(exampleRepresentationOfQueryObject),
+                query: qs.stringify(exampleRepresentationOfQueryObject, {arrayFormat: this.provider.queryArrayFormat || 'brackets'}),
```

Comment:
this is different to default arrayFormat of qs.striginfy (as visible on some snapshots in diff). Default value is `indices` - could you change our default to match qs.stringify default?